### ISSUE TITLE
Add Link to the uploaded artifact file

### ISSUE THREAD

**Issue by ahmetkucuk** [zgosalvez/github-actions-report-lcov#172]
It would be nice if the comment has a link to the artifact uploaded to Github.

In addition, we might return artifact-id that we receive from uploading an artifact. So that we can programmatically decide where to show/serve it.

**Comment by sdankel** [comment]
I agree, this would be very nice! Even better would be an option to have the report attached to the comment.
